docs/REST API/rest-api.md

```diff
@@ -4991,6 +4995,78 @@ Content-Length: <length>
 Body: <blob>
 ```
 
+### Lock host
+
+Sends a command to lock the specified macOS, Linux, or Windows host. The host is locked once it comes online.
+
+`GET /api/v1/fleet/mdm/lock/:identifier`
```

Comment:
I kinda prefer the look of `/mdm/hosts/:identifier/lock`, but I feel like since we already have `mdm/hosts/:id` urls it'd be weird to have a varying dynamic parameters come after `/hosts/`. So I think this pattern is our best option for these endpoints.